demisto_sdk/dev_tools/linter.py

```diff
@@ -79,7 +79,11 @@ def remove_common_server_python(self):
 
     def run_dev_packages(self) -> int:
         # load yaml
-        yml_path = glob.glob(self.project_dir + '/*.yml')[0]
+        try:
+            yml_path = glob.glob(self.project_dir + '/*.yml')[0]
```

Comment:
I think before we access the `[0]` item we should check if it returned any results instead of try/catch.